.github/workflows/release-draft-binaries.yml

```diff
@@ -433,38 +431,16 @@ jobs:
         with:
           ref: ${{ github.event.inputs.release_branch }}
 
-      - name: Install Rust toolchain
-        run: |
-          # This will allow us update to rust version indicated in our rust-toolchain.toml file
-          rustup show
-          rustup target add ${{ matrix.job.target }}
-
-      - name: Install Cross
-        if: matrix.job.use-cross == true
-        run: cargo install --version 0.2.4 cross
-
-      - shell: bash
-        if: matrix.job.target == 'ubuntu-20.04'
+      - name: Install Nix
```

Comment:
Let's use this third party action to install Nix https://github.com/DeterminateSystems/nix-installer-action

Reply:
Done